### DISCUSSION TITLE
How does one obtain the 95% CI of effect sizes?

### DISCUSSION THREAD

**Discussion by pcrxn** [FrederickHuangLin/ANCOMBC#34] [Q&A] [answered]
Hello! I was wondering how to obtain the 95% simultaneous confidence intervals of effect sizes, as described in the ANCOM-BC paper?

> "ANCOM-BC is the first method in the literature that can not only identify differentially abundant taxa while controlling the FDR for multiple testing, it also provides 95% simultaneous confidence intervals for the mean DA of each taxon in the two experimental groups."

Thank you!

Liam

**Comment by FrederickHuangLin** [comment, marked as answer]
Hi,

If the multiple comparison correction method you chose is "Bonferroni", the construction of simultaneous confidence interval is straightforward by replacing the alpha level with adjusted alpha level (alpha/# tests), and obtain the Wald type of confidence level using the estimated coefficient, SE, and adjusted alpha. However, if you would like to use other correction methods such as BH procedure, the construction of simultaneous confidence interval is currently not available.

Best,
Huang

**Reply by pcrxn** [reply]
Thank you Huang, this answered my question!
